src/components/progress-tracker/progress-tracker.test.js

```diff
@@ -169,4 +181,135 @@ context("Tests for ProgressTracker component", () => {
+      (size) => {
+        CypressMountWithProviders(<ProgressTrackerComponent size={size} />);
+
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each([150, 350, 550])(
+      "should check the accessibility when component is rendered %s px length",
+      (length) => {
+        CypressMountWithProviders(
+          <ProgressTrackerComponent length={`${length}px`} />
+        );
+
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each([12, 47, 100])(
+      "should check the accessibility when component is rendered with %s% of progress",
+      (progress) => {
+        CypressMountWithProviders(
+          <ProgressTrackerComponent progress={progress} />
+        );
+
+        cy.checkAccessibility();
+      }
+    );
+
+    it("should check the accessibility when component is rendered with error prop", () => {
+      CypressMountWithProviders(
+        <ProgressTrackerComponent error showDefaultLabels progress={35} />
+      );
+
+      cy.checkAccessibility();
+    });
+
+    it.each([true, false])(
+      "should check the accessibility when component is rendered with showDefaultLabels is set to %s",
+      (boolean) => {
+        CypressMountWithProviders(
+          <ProgressTrackerComponent showDefaultLabels={boolean} />
+        );
+
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each([CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS])(
+      "should check the accessibility when component is rendered with currentProgressLabel is set to %s",
+      (currentProgressLabel) => {
+        CypressMountWithProviders(
+          <ProgressTrackerComponent
+            currentProgressLabel={currentProgressLabel}
+          />
+        );
+
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each([CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS])(
+      "should check the accessibility when component is rendered with maxProgressLabel is set to %s",
+      (maxProgressLabel) => {
+        CypressMountWithProviders(
+          <ProgressTrackerComponent maxProgressLabel={maxProgressLabel} />
+        );
+
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each([CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS])(
+      "should check the accessibility when component is rendered with customValuePreposition is set to %s",
+      (customValuePreposition) => {
+        CypressMountWithProviders(
+          <ProgressTrackerComponent
+            customValuePreposition={customValuePreposition}
+            showDefaultLabels
+          />
+        );
+
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each(["top", "bottom"])(
+      "should check the accessibility when component is rendered with labelsPosition is set to %s",
+      (labelsPosition) => {
+        CypressMountWithProviders(
+          <ProgressTrackerComponent labelsPosition={labelsPosition} />
+        );
+
+        cy.checkAccessibility();
+      }
+    );
+  });
 });
```

Comment:
Do we not need to make some changes to stories.tsx file like putting "export const ProgressTrackerComponent" from the current test.js file into that tsx file?

Reply:
this component is still not refactored to `ts` so there is no point of doing that